test/unit/app/messaging/process-backoffice.test.js

```diff
@@ -0,0 +1,70 @@
+const processBackOffice = require('../../../../app/messaging/process-backoffice')
+const { backOfficeResponseMsgType, backOfficeResponseQueue } = require('../../../../app/config')
+const states = require('../../../../app/messaging/states')
+
+jest.mock('../../../../app/lib/send-email')
+const sendMessage = require('../../../../app/messaging/send-message')
+jest.mock('../../../../app/messaging/send-message')
+const applicationRepository = require('../../../../app/repositories/application-repository')
+jest.mock('../../../../app/repositories/application-repository')
+
+describe(('Store backOffice in database'), () => {
```

Comment:
Remove unnecessary braces and update description as this is about processing the message rather than storing the back-office in the database!